src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.xml

```diff
@@ -50,6 +50,10 @@
             <tag name="texter.transport_factory" />
         </service>
 
+        <service id="notifier.transport_factory.smsapi" class="Symfony\Component\Notifier\Bridge\Smsapi\SmsapiTransportFactory" parent="notifier.transport_factory.abstract">
+            <tag name="texter.transport_factory" />
+        </service>
```

Comment:
Can you move it to the new PHP config file?

Reply:
Yes, I've done